src/main/java/sirius/biz/tenants/TenantSearchProvider.java

```diff
@@ -69,10 +68,13 @@ public void query(String query, int maxResults, Consumer<OpenSearchResult> resul
             openSearchResult.withURL("/tenant/" + tenant.getIdAsString());
 
             if (currentUser.hasPermission(TenantUserManager.PERMISSION_SELECT_TENANT)) {
-                openSearchResult.withQuickAction(new QuickAction().withLabel(NLS.get("TenantController.select"))
-                                                                  .withIcon("fa fa-factory")
-                                                                  .withUrl("/tenants/select/"
-                                                                           + tenant.getIdAsString()));
+                openSearchResult.withTemplateFromCode("""
+                                                              <i:arg name="tenant" type="sirius.biz.tenants.Tenant"/>
+                                                                   
+                                                              @Formatter.create("[${zip}][ ${city}]").set("zip", tenant.getTenantData().getAddress().getZip()).set("city", tenant.getTenantData().getAddress().getCity()).smartFormat() 
```

Comment:
being html this could be 
```suggestion
                                                              @tenant.getTenantData().getAddress().getZip() @tenant.getTenantData().getAddress().getCity()
```